docs/topics/numbers.md

```diff
@@ -0,0 +1,295 @@
+[//]: # (title: Numbers)
+
+## Integer types
+
+Kotlin provides a set of built-in types that represent numbers.  
+For integer numbers, there are four types with different sizes and, hence, value ranges.
+
+| Type	 |Size (bits)| Min value| Max value|
+|--------|-----------|----------|--------- |
+| `Byte`	 | 8         |-128      |127       |
+| `Short`	 | 16        |-32768    |32767     |
+| `Int`	 | 32        |-2,147,483,648 (-2<sup>31</sup>)| 2,147,483,647 (2<sup>31</sup> - 1)|
+| `Long`	 | 64        |-9,223,372,036,854,775,808 (-2<sup>63</sup>)|9,223,372,036,854,775,807 (2<sup>63</sup> - 1)|
+
+If you initialize a variable with an integer value not exceeding the maximum value of `Int`, the compiler will 
```

Comment:
"not exceeding the range of" or something like that, because it could exceed (be greater than) max value or (be less than) min value.

Reply:
Fixed